components/renderer/CaseStudy.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import LinkWrapper from '@components/LinkWrapper';
+import Image from 'next/image';
+import classNames from 'classnames';
+import { TypePageCaseStudy, TypePageFields } from '@types';
+import * as Contentful from 'contentful';
+
+type CaseStudyPropTypes = {
+    pageCaseStudy: Contentful.Entry<TypePageFields>;
```

Comment:
`Contentful.Entry<TypePageFields>` is the same as `TypePage`. `TypePage` should be used because it means the `Contentful` import isn't necessary.